buidl/mnemonic.py

```diff
@@ -34,2082 +38,63 @@ def secure_mnemonic(num_bits=256, extra_entropy=0):
-    "truth",
-    "try",
-    "tube",
-    "tuition",
-    "tumble",
-    "tuna",
-    "tunnel",
-    "turkey",
-    "turn",
-    "turtle",
-    "twelve",
-    "twenty",
-    "twice",
-    "twin",
-    "twist",
-    "two",
-    "type",
-    "typical",
-    "ugly",
-    "umbrella",
-    "unable",
-    "unaware",
-    "uncle",
-    "uncover",
-    "under",
-    "undo",
-    "unfair",
-    "unfold",
-    "unhappy",
-    "uniform",
-    "unique",
-    "unit",
-    "universe",
-    "unknown",
-    "unlock",
-    "until",
-    "unusual",
-    "unveil",
-    "update",
-    "upgrade",
-    "uphold",
-    "upon",
-    "upper",
-    "upset",
-    "urban",
-    "urge",
-    "usage",
-    "use",
-    "used",
-    "useful",
-    "useless",
-    "usual",
-    "utility",
-    "vacant",
-    "vacuum",
-    "vague",
-    "valid",
-    "valley",
-    "valve",
-    "van",
-    "vanish",
-    "vapor",
-    "various",
-    "vast",
-    "vault",
-    "vehicle",
-    "velvet",
-    "vendor",
-    "venture",
-    "venue",
-    "verb",
-    "verify",
-    "version",
-    "very",
-    "vessel",
-    "veteran",
-    "viable",
-    "vibrant",
-    "vicious",
-    "victory",
-    "video",
-    "view",
-    "village",
-    "vintage",
-    "violin",
-    "virtual",
-    "virus",
-    "visa",
-    "visit",
-    "visual",
-    "vital",
-    "vivid",
-    "vocal",
-    "voice",
-    "void",
-    "volcano",
-    "volume",
-    "vote",
-    "voyage",
-    "wage",
-    "wagon",
-    "wait",
-    "walk",
-    "wall",
-    "walnut",
-    "want",
-    "warfare",
-    "warm",
-    "warrior",
-    "wash",
-    "wasp",
-    "waste",
-    "water",
-    "wave",
-    "way",
-    "wealth",
-    "weapon",
-    "wear",
-    "weasel",
-    "weather",
-    "web",
-    "wedding",
-    "weekend",
-    "weird",
-    "welcome",
-    "west",
-    "wet",
-    "whale",
-    "what",
-    "wheat",
-    "wheel",
-    "when",
-    "where",
-    "whip",
-    "whisper",
-    "wide",
-    "width",
-    "wife",
-    "wild",
-    "will",
-    "win",
-    "window",
-    "wine",
-    "wing",
-    "wink",
-    "winner",
-    "winter",
-    "wire",
-    "wisdom",
-    "wise",
-    "wish",
-    "witness",
-    "wolf",
-    "woman",
-    "wonder",
-    "wood",
-    "wool",
-    "word",
-    "work",
-    "world",
-    "worry",
-    "worth",
-    "wrap",
-    "wreck",
-    "wrestle",
-    "wrist",
-    "write",
-    "wrong",
-    "yard",
-    "year",
-    "yellow",
-    "you",
-    "young",
-    "youth",
-    "zebra",
-    "zero",
-    "zone",
-    "zoo",
-]
+def get_bip39_index(word):
+    if not BIP39_WORDS:
+        __init__()
+    return BIP39_LOOKUP[word]
 
 
-__init__()
+def get_bip39_word(index):
+    """Get the word at a particular index"""
+    # lazy load
+    if not BIP39_WORDS:
+        __init__()
+    return BIP39_WORDS[index]
+
+
+def __init__():
+    # load word from file and make a lookup table
+    word_file = path.join(path.dirname(__file__), "bip39_words.txt")
+    global BIP39_WORDS, BIP39_LOOKUP
```

Comment:
Is a global needed? Can't this code block just run every time this file is loaded (outside the `_init_()`.

I don't have a lot of experience with globals and thought the best practice was to avoid them unless there was a strong reason. I could be wrong though.